Source/src/components/ComponentAnimation.cpp

```diff
@@ -167,6 +187,8 @@ void Hachiko::ComponentAnimation::DrawGui()
                 App->resources->ReleaseResource(state_machine);
                 std::vector<UID> uids = App->resources->CreateAsset(Resource::Type::STATE_MACHINE, auxiliary_name);
                 state_machine = static_cast<ResourceStateMachine*>(App->resources->GetResource(Resource::Type::STATE_MACHINE, uids[0]));
+                // Once a new state machine is added a "none" clip is added to it
+                state_machine->AddClip("none", 0, false);
```

Comment:
Same with the "none", let's make it into a `constexpr const char*` for better readability and ease of change.